src/pages/index.js

```diff
@@ -8,61 +8,61 @@ import styles from "./styles.module.css";
 
 const features = [
   {
-    title: "I'm new here, and want to get started",
-    imageUrl: "img/confused-person.png",
+    title: "Get started",
+    imageUrl: "img/home-get-started.png",
     url: "/docs/guides",
     description:
-      "Visit our guides section to learn more about creating an account, modeling your first process, orchestrating human and service tasks, and more!",
+      "New to Camunda 8? Create an account and start modeling your first process.",
   },
   {
-    title: "I'm looking for a particular component",
-    imageUrl: "img/homegrown-workflow.png",
+    title: "Components",
+    imageUrl: "img/home-components.png",
     url: "/docs/components/",
     description:
-      "Find product manual content for each component in Camunda 8. Together, these components comprise the Camunda 8 SaaS experience.",
+      "Explore components such as Modeler, Zeebe, Connectors, Operate, Tasklist, and Optimize.",
```

Comment:
I would add Console before Modeler, and I'm having a hard time naming Connectors a component -- let's remove it here until I can define in my work on our [Camunda intro docs](https://github.com/camunda/camunda-docs/issues/4011).

Reply:
Changed to: "Explore components such as Console, Modeler, Zeebe, Operate, Tasklist, and Optimize."